eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -0,0 +1,28 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# This is to help with testing the release pipeline.
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
+$latestTags = git tag -l "Azure.Template*beta*"
+$semVars = @()
+
+$changeLogFile = "${PSScriptRoot}\..\..\sdk\template\Azure.Template\CHANGELOG.md"
+
+Foreach ($tags in $latestTags)
+{
+  $semVars += $tags.Replace("Azure.Template_", "")
+}
+
+$semVarsSorted = [AzureEngSemanticVersion]::SortVersionStrings($semVars)
+LogDebug "Last Published Version $($semVarsSorted[0])"
+
+$newVersion = [AzureEngSemanticVersion]::ParseVersionString($semVarsSorted[0])
+$newVersion.IncrementAndSetToPrerelease()
+LogDebug "Version to publish [ $($newVersion.ToString()) ]"
+
+&"${PSScriptRoot}/Update-PkgVersion.ps1" -ServiceDirectory "template" `
+-PackageName 'Azure.Template' -PackageDirName "Azure.Template" -NewVersionString $newVersion.ToString() `
```

Comment:
Similar to the other PR's I'd like to see if we can use the change log operations for adding the notes.